examples/31-set-tags-for-entries.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (migration) {
+  migration.setTagsForEntries({
+    contentType: 'article',
+    from: ['title'],
+    setTagsForEntry: (fields, entryTags, apiTags) => {
+      const newTag = apiTags.find(tag => tag.sys.id === 'old');
```

Comment:
I know it's just an example, but I', a bit confused why `newTag` finds the tag with the ID `old` in the example 🙂

Reply:
fair point :)